pkg/theiactl/commands/result.go

```diff
@@ -49,9 +50,34 @@ $ theiactl policyreco result --id e998433e-accb-4888-9fc8-06563f073e86
 		if err != nil {
 			return fmt.Errorf("couldn't create k8s client using given kubeconfig, %v", err)
 		}
-		// TODO: setup port forwarder using clientset.
-		// TODO: get the recommendation result from ClickHouse using recoID
-		fmt.Printf("recoID: %v, clientset: %v\n", recoID, clientset)
+		err = CheckClickHousePod(clientset)
```

Comment:
I think the aim of this check is to prevalidate that we have a running clickhouse pod.
I do not think it adds much overhead, and it could be fine for the time being.

In the future I think we should have a more precise solution for validating the health status of the flow visibility solution